# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/common/mod.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/common/mod.rs b/third_party/rust/spirv-cross-internal/tests/common/mod.rs
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/common/mod.rs
+++ /dev/null
@@ -1,9 +0,0 @@
-#[allow(clippy::cast_ptr_alignment)]
-pub fn words_from_bytes(buf: &[u8]) -> &[u32] {
-    unsafe {
-        std::slice::from_raw_parts(
-            buf.as_ptr() as *const u32,
-            buf.len() / std::mem::size_of::<u32>(),
-        )
-    }
-}